recipes/default.rb

```diff
@@ -22,3 +22,26 @@
 # Package
 package 'docker-engine'
 
+# Service configuration
+template '/lib/systemd/system/docker.service' do
```

Comment:
Instead of the `template` resource, you could also use the `systemd_service`, like we do here: https://github.com/TYPO3-cookbooks/gerrit/blob/71786d626e3b2f2e8020bf421d9a12b3affff6bc/recipes/_system.rb#L53-L73

I'm still fine with keeping a template.